src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -87,12 +87,11 @@ case class WorkflowActor private(id: WorkflowId,
   private def startRunnableCalls(): Unit = {
     for {
       runnableCalls <- (storeActor ? FindRunnableCalls).mapTo[Iterable[Call]]
-      _ <- storeActor ? PrepareToStartCalls(runnableCalls)
     } yield {
-      if (runnableCalls.nonEmpty) {
-        log.info("Starting calls: " + runnableCalls.map {_.name}.toSeq.sorted.mkString(", "))
+      runnableCalls foreach {call =>
+        log.info("Starting call: " + call.name)
```

Comment:
The previous logic enabled the test to confirm that multiple calls became runnable at the same time, which this logic apparently does not.